DirectProgramming/DPC++FPGA/Tutorials/Features/ac_fixed/README.md

```diff
@@ -259,33 +259,50 @@ Scroll down on the Summary page of the report and expand the section titled **Co
    ./ac_fixed.fpga             # Linux
    ```
 
-### Example of Output
+### Example of Output on Emulator
 
 ```txt
+1. Testing Constructing ac_fixed from float or ac_fixed:
 Constructed from float:         3.6416015625
 Constructed from ac_fixed:      3.6416015625
 
-MAX DIFF for ac_fixed<10, 3, true>:  0.0078125
-MAX DIFF for float:                  9.53674e-07
-
-result(fixed point): 1
-result(float):       1
-
-result(fixed point): 0.992188
-result(float):       1
-
-result(fixed point): 1
-result(float):       1
-
-result(fixed point): 1
-result(float):       1
-
-result(fixed point): 0.992188
-result(float):       1
+2. Testing calculation with float or ac_fixed math functions:
+MAX DIFF (quantum) for ac_fixed<10, 3, true>:   0.0078125
+MAX DIFF for float:                             9.53674e-07
+
+Input 0:                        -0.80799192
+result(fixed point):            1
+difference(fixed point):        0
+result(float):                  1
+difference(float):              0
+
+Input 1:                        -2.099829
+result(fixed point):            0.9921875
+difference(fixed point):        0.0078125
+result(float):                  0.99999994
+difference(float):              5.9604645e-08
+
+Input 2:                        -0.74206626
+result(fixed point):            1
+difference(fixed point):        0
+result(float):                  1
+difference(float):              0
+
+Input 3:                        -2.3321707
+result(fixed point):            1
+difference(fixed point):        0
+result(float):                  1
+difference(float):              0
+
+Input 4:                        1.1432415
+result(fixed point):            0.9921875
+difference(fixed point):        0.0078125
+result(float):                  0.99999994
+difference(float):              5.9604645e-08
 
 PASSED: all kernel results are correct.
 ```
 
 ### Discussion of Results
 
-You can obtain a smaller hardware footprint for your kernel by ensuring that the `ac_fixed` numbers are constructed from `float` or `double` numbers outside the kernel. Additionally, you can trade-off mathematical operation accuracy for a more resource efficient design by using the `ac_fixed` math library functions.
\ No newline at end of file
+You can obtain a smaller hardware footprint for your kernel by ensuring that the `ac_fixed` numbers are constructed from `float` or `double` numbers outside the kernel. Additionally, you can trade-off mathematical operation accuracy for a more resource efficient design by using `ac_fixed` types of a smaller size and the corresponding `ac_fixed` math library functions.
```

Comment:
"of a smaller size" is not quite correct as you can use a larger fixed-point format than float and have arithmetic operators that use fewer resources than a floating point operator.
I would rephrase as follows: 
 "Additionally, by using the `ac_fixed` types and math library functions, you can use an even smaller fixed point format to trade-off even more accuracy for a more resource efficient design if your application requirements allows for it."